pkg/pillar/cmd/volumemgr/handlework.go

```diff
@@ -367,3 +184,47 @@ func casIngestWorker(ctxPtr interface{}, w worker.Work) worker.WorkResult {
 	}
 	return result
 }
+
+// processVolumeWorkResult handle the work result that was a volume action
+func processVolumeWorkResult(ctxPtr interface{}, res worker.WorkResult) error {
+	ctx := ctxPtr.(*volumemgrContext)
+	d := res.Description.(volumeWorkDescription)
+	updateVolumeStatus(ctx, d.status.VolumeID)
+	return nil
+}
+
+// processCasIngestWorkResult handle the work result that was a cas ingestion
+func processCasIngestWorkResult(ctxPtr interface{}, res worker.WorkResult) error {
+	ctx := ctxPtr.(*volumemgrContext)
+	d := res.Description.(casIngestWorkDescription)
+	// this might have changed, so we want to be careful about passing it; always look it up
+	updateContentTreeByID(ctx, d.status.Key())
+	return nil
+}
+
+// lookupCasIngestWorkResult get the result
```

Comment:
I'd expect a function named 'lookupX' to not change state, but here we do a Pop. So should we name the functions which go a Pop 'getX' instead?

Reply:
The comment makes sense. I am ok with that.

Reply:
Took care of the renaming.